packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/ProjectController.java

```diff
@@ -466,19 +471,19 @@ public ResponseEntity<ProjectAsset> createAsset(
 					@Content(mediaType = "application/json", schema = @io.swagger.v3.oas.annotations.media.Schema(implementation = UUID.class)) }),
 			@ApiResponse(responseCode = "204", description = "User may not have permission to this project", content = @Content),
 			@ApiResponse(responseCode = "500", description = "Error finding project", content = @Content) })
-	@DeleteMapping("/{id}/assets/{asset-type}/{asset-id}")
+	@DeleteMapping("/{id}/assets/{asset-type}/{project-asset-id}")
 	@Secured(Roles.USER)
 	public ResponseEntity<ResponseDeleted> deleteAsset(
 			@PathVariable("id") final UUID projectId,
 			@PathVariable("asset-type") final AssetType type,
-			@PathVariable("asset-id") final UUID assetId) {
+			@PathVariable("project-asset-id") final UUID projectAssetId) {
```

Comment:
It _is_ different. The `asset-id` in the `POST` refers to the asset you are adding to the project. Once its created, it gets its own unique id in the `project-asset` table, which is what the `project-asset-id` refers to.

It's pretty unintuitive, and if you would like I could instead change the `DELETE` to lookup and delete by the original asset-id instead.

Reply:
I think that would be preferable, because on the front-end we might delete those asset on places where we are not in a project. i.e. a search result, etc.

Reply:
Done and done!